### PR TITLE
Pr for hw 15

### PR DESCRIPTION
**Pull request by landriiuk**

### CODE REVIEW

File: cypress/integration/sample_spec.js [hunk 1]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+
+  
+  
+ 
+
+  describe('HW 15', () => {
+    it('Create a new user', () => {
+      
+      
+        cy.visit('https://demoqa.com/webtables', { pageloadtimeout: 10000 })
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: I wrote comment in [HW14](https://github.com/Pecode-AQA-Intership/A_Davydiuk/issues/2) , check it pls.
pageloadtimeout better to add into cypress.json file

File: cypress/integration/sample_spec.js [hunk 2]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+
+        cy.get('#submit').click()
+
+        
+    });
+
+    it('Verify that user was added', () => {
+            cy.get('.ReactTable').should('contain', 'Harry');
+           
+
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: empty lines

File: cypress/integration/sample_spec.js [hunk 3]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+        cy.get('#submit').click()
+
+        
+    });
+
+    it('Verify that user was added', () => {
+            cy.get('.ReactTable').should('contain', 'Harry');
+           
+        
+ })
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: missed ";"

File: cypress/integration/sample_spec.js [hunk 4]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+        .clear()
+        .type(46)
+        .should('have.value', 46);
+
+        cy.get('#salary').click()
+        .clear()
+        .type(3000)
+        .should('have.value', 3000);
+
+
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: And here lines

File: cypress/integration/sample_spec.js [hunk 5]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+        cy.get('#submit').click()
+ });
+
+    it('Check that all data was  edited', () => {
+        cy.get('.ReactTable').should('contain', 'Tom');
+        cy.get('.ReactTable').should('contain', 'Riddle');
+        cy.get('.ReactTable').should('contain', 'qwerty@gmail.com');
+        cy.get('.ReactTable').should('contain', '3000');
+        cy.get('.ReactTable').should('contain', '46');
+        cy.get('.ReactTable').should('contain', 'Depaetment 007')
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: U can use within here, or add verification using "and()"

File: cypress/integration/sample_spec.js [hunk 6]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+    it('Delete user from the table and check that user was deleted', () => {
+        cy.get('#delete-record-4 > svg').click()
+        cy.get('.ReactTable').should('not.contain', 'Tom')
+    })
+
+
+    it('Check searching feature', () => {
+        cy.get('#addNewRecordButton', { timeout: 10000 }).click();
+        
+
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: Same here about the lines

File: cypress/integration/sample_spec.js [hunk 7]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+    it('Implement sorting tests', () => {
+        
+       cy.contains('First Name').should('exist').click()
+       cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(1)")
+
+       .then(tableBody => {
+        let sortNamefromSite = [];
+        sortNamefromSite.push(tableBody.text());
+        cy.log(sortNamefromSite)
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: Pls delete logs from your code

File: cypress/integration/sample_spec.js [hunk 8]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+        let sortNameHere = [];
+        sortNameHere.push(tableBody.text())
+        cy.log(sortNameHere)
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: And here

File: cypress/integration/sample_spec.js [hunk 9]
```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
...
+        sortDepartmentHere.push(tableBody6.text())
+        cy.log(sortDepartmentHere)
+        expect(sortDepartmentHere.sort()).to.eql(sortDepartmentfromSite)
+
+
+    }) 
+
+      
+}) 
+
```
Comment (landriiuk) [inline cypress/integration/sample_spec.js]: U are able to create a few reusable functions, u have a lot of copy pasted code

**Comment by landriiuk** [comment]
@adavydiuk Fix comments pls till tomorrow!

File: cypress/integration/sample_spec.js (hunk 1 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 2 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 3 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 4 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 5 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 6 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 7 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 8 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: fixed

File: cypress/integration/sample_spec.js (hunk 9 above)
Comment (adavydiuk) [reply cypress/integration/sample_spec.js]: can you share with me some links how to do that?

**Comment by landriiuk** [comment]
Score 9/10

U are able to merge this branch into master

**Pull request merged**